packages/webviz-core/src/util/hooks.js

```diff
@@ -1,38 +0,0 @@
-// @flow
```

Comment:
These probably should not get deleted (even though they're not currently used in this repo). Or do you plan on moving them out of webviz-core?

Reply:
We discussed. Think @esthersweon  is going to delete the file from webviz-core  as well in her current PR. This will help testing her change in CI.

Reply:
Yep - this is the PR that will delete from webviz-core in the internal repo (as well as change all references to these deleted exports): https://github.robot.car/cruise/webviz/pull/1554